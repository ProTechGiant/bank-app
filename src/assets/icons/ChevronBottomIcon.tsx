import Svg, { Path } from "react-native-svg";

import type { IconProps } from "@/assets/icons";

export function ChevronBottomIcon({ color = "#FF5B00", height = 24, width = 24 }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M12 1.70687L10.59 0.296875L6 4.87687L1.41 0.296875L-6.16331e-08 1.70687L6 7.70687L12 1.70687Z"
        fill={color}
      />
    </Svg>
  );
}
