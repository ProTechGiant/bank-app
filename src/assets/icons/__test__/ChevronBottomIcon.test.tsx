import { create } from "react-test-renderer";

import { ChevronBottomIcon } from "../ChevronBottomIcon";

describe("icon/ChevronBottomIcon", () => {
  it("renders a ChevronBottomIcon", () => {
    const result = create(<ChevronBottomIcon />);

    expect(result).toMatchSnapshot();
  });
});
