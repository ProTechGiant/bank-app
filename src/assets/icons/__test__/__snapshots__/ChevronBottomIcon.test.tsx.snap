// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/ChevronBottomIcon renders a ChevronBottomIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="24"
  bbWidth="24"
  fill="none"
  focusable={false}
  height={24}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 24,
        "width": 24,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width={24}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M7.41 8.29688L12 12.8769L16.59 8.29688L18 9.70687L12 15.7069L6 9.70687L7.41 8.29688Z"
      fill={
        {
          "payload": 4294925056,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
